js/5etools-deities.js

```diff
@@ -0,0 +1,76 @@
+function d20plusDeities () {
+	d20plus.deities = {};
+
+	// Import Object button was clicked
+	d20plus.deities.button = function () {
+		const url = $("#import-deities-url").val();
+
+		if (url && url.trim()) {
+			DataUtil.loadJSON(url).then((data) => {
+				d20plus.importer.addBrewMeta(data._meta);
+				d20plus.importer.showImportList(
+					"deity",
+					data.deity,
+					d20plus.deities.handoutBuilder,
+				);
+			});
+		}
+	};
+
+    d20plus.deities.handoutBuilder = function (data, overwrite, inJournals, folderName, saveIdsTo, options) {
+		// make dir
+		const folder = d20plus.journal.makeDirTree(`Deities`, folderName);
+		const path = ["Deities", ...folderName, data.name];
+
+		// handle duplicates/overwrites
+		if (!d20plus.importer._checkHandleDuplicate(path, overwrite)) return;
+
+		const name = data.name;
+		d20.Campaign.handouts.create({
+			name: name,
```

Comment:
deity names on the site are rendered as (effectively)
```
`${deity.name}${deity.title ? `, ${deity.title.toTitleCase()}` : ""}`
```
so I would recommend doing the same here